scripts/regenerate.sh

```diff
@@ -22,9 +22,9 @@ function finish {
 }
 trap finish EXIT
 
-export GOBIN=${WORKDIR}/bin
-export PATH=${GOBIN}:${PATH}
-mkdir -p ${GOBIN}
+GOBIN="${WORKDIR}"/bin
```

Comment:
Why is GOBIN not `export`ed now?

Reply:
cz we are already making sure the binary are installed [here](https://github.com/grpc/grpc-go/pull/7064/files#diff-0f7c4e0b21e1ac633cff1e9f33a667bbddac061a3c39e18d88d883632e73e46bR31)